packages/react-components/src/components/datatree-output-component.tsx

```diff
@@ -154,4 +160,38 @@ export class DataTreeOutputComponent extends AbstractOutputComponent<AbstractOut
         // fix Warning: Can't perform a React state update on an unmounted component
         this.setState = (_state, _callback) => undefined;
     }
+
+    protected async fetchSelectionData(): Promise<void> {
+        if (this.props.selectionRange) {
+            const origXYTree = cloneDeep(this.state.xyTree);
+            origXYTree.sort((a,b)=> (a.id > b.id) ? 1 : -1);
+
+            let payload: any;
+            if (Number(this.props.selectionRange.getStart()) < Number(this.props.selectionRange.getEnd())) {
```

Comment:
I think this might give the wrong result if both values are very large and very close, due to the loss of precision. You can compare the bigint values directly?

Reply:
> I think this might give the wrong result if both values are very large and very close, due to the loss of precision. You can compare the bigint values directly?

Didn't know that, I'll make the change